docs/sdk/api-reference.md

```diff
@@ -3730,10 +2824,10 @@ compute_cluster.delete_nodepools(
 )
 ```
 
-### ComputeCluster._process_nodepool_config()
+### ComputeCluster._process_nodepool_config
```

Comment:
This is a internal method. Remove it

Reply:
As this is an internal method i have removed this from the doc